PhotoPlusBackend/photoplus/src/main/java/pl/polsl/photoplus/services/controllers/AbstractModelService.java

```diff
@@ -94,7 +94,8 @@ public AbstractModelService(final R entityRepository)
     @Override
     public T getSingleObject(final String code)
     {
-        return getDtoFromModel(findByCodeOrThrowError(code, "FIND SINGLE"));
```

Comment:
i think that getSingleObject should throw an error when object with given code is not found